doc/qrexec-remotevm.rst

```diff
@@ -0,0 +1,173 @@
+      This would allow unexpected or misconfigured connections to be identifiable and managed rather than outright rejected.
+
+Particular case: registering a RemoteVM not being on a QubesOS host
+--------------------------------------------------------------------
+
+The general design also allows defining and executing a call to a RemoteVM that is not a “qube” on a Qubes OS host.
+By installing ``qrexec-client-vm`` on standard Linux or Windows machines, you can execute RPC on these platforms.
+This is particularly interesting if you need to interact with, for example, KVM virtual machines, RaspberryPi devices,
+standard computers with extensive resources, or computers with architectures different from those currently supported by Qubes OS.
+It requires adapting the ``qrexec`` codebase to manage policies directly on the machine where ``qrexec-client-vm`` runs, rather than in ``dom0``.
+
+In this particular case, assume that there is a LocalVM named ``Local-Relay`` and that ``Remote-Relay`` is running as a compatible
+``qrexec-client-vm`` on a non-Qubes OS host. Since a RelayVM is itself a RemoteVM, simply define ``Remote-Relay`` as a standard
+RemoteVM with the name ``Remote-Qube``, resulting in the following sequence diagram:
+
+.. image:: _static/particular.png
+
+SSH-based communication between Local‑Relay and Remote‑Relay
+------------------------------------------------------------
+
+Prerequisites
+~~~~~~~~~~~~~
+
+- SSH key-based authentication is configured between all involved hosts (Local‑Relay, Remote‑Relay, and any intermediate jump hosts).
+- (Optional) Configure any necessary proxies or jump hosts by following the OpenSSH Cookbook guidelines: Proxies and Jump Hosts <https://en.wikibooks.org/wiki/OpenSSH/Cookbook/Proxies_and_Jump_Hosts>.
+
+Example Transport RPC: ``qubesair.SSHProxy``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Below is an example Bash script implementing the ``qubesair.SSHProxy`` transport RPC. It parses a destination and service, looks up the remote Qube name, and invokes ``qrexec-client-vm`` over SSH.
+
+.. code-block:: bash
+
+   #!/bin/bash
+
+   set -euo pipefail
+
+   if [[ -z "${1:-}" ]]; then
+       echo "Usage: $0 <target+service>"
+       exit 1
+   fi
+
+   # Extract target and service
+   IFS='+' read -r target service <<< "$1"
+
+   # Validate extracted values
+   if [[ -z "$target" || -z "$service" ]]; then
+       echo "Error: Invalid input format. Expected 'target+service'."
+       exit 1
+   fi
+
+   # Retrieve the remote Qube identifier
+   remote_qube="$(qubesdb-read "/remote/$target" 2>/dev/null || true)"
+
+   if [[ -z "$remote_qube" ]]; then
+       echo "Error: Could not read remote qube for target '$target'."
+       exit 1
+   fi
+
+   # Forward the qrexec call via SSH
+   ssh "$remote_qube" qrexec-client-vm --source-qube="$QREXEC_REMOTE_DOMAIN" "$remote_qube" "$service"
+
+Workflow Overview
```

Comment:
IMO better to put description before the example. The description is (should be) what the service should do, and then example based on SSH. It should be possible to write also other services based on this description.